tests/lib/modem_trace/modem_trace_rtt/testcase.yaml

```diff
@@ -0,0 +1,11 @@
+tests:
+  lib.modem_trace_rtt:
+    platform_allow: qemu_cortex_m3
+    integration_platforms:
+      - qemu_cortex_m3
+    tags: modem_trace
+    extra_configs:
+      # The promptless symbol HAS_SEGGER_RTT is defined in the tests' Kconfig file.
+      - CONFIG_USE_SEGGER_RTT=y
+      - CONFIG_SEGGER_RTT_MAX_NUM_UP_BUFFERS=3
+      - CONFIG_SEGGER_RTT_MAX_NUM_DOWN_BUFFERS=3
```

Comment:
Those are not needed as this is already the default when `CONFIG_USE_SEGGER_RTT=y`
https://github.com/nrfconnect/sdk-zephyr/blob/eb0141519f9c278ed2c255f461b8eb82bd3566df/drivers/debug/Kconfig.rtt#L24-L30
```
config SEGGER_RTT_MAX_NUM_UP_BUFFERS
	int "Maximum number of up-buffers"
	default 3


config SEGGER_RTT_MAX_NUM_DOWN_BUFFERS
	int "Maximum number of down-buffers"
	default 3
```

Reply:
Removed these in the last push.